modules/samples-operator-overview.adoc

```diff
@@ -47,3 +47,18 @@ the following upon deletion:
 
 Upon deletion of the samples resource, the Samples Operator recreates the
 resource using the default configuration.
+
+[id="samples-operator-bootstrapped"]
+== Samples Operator bootstrapped as removed
+
+Certain circumstances will result in the Samples Operator bootstrapping itself as removed including:
+
+* If the Samples Operator cannot reach link:registry.redhat.io[registry.redhat.io] after three minutes on initial start-up after installation.
+* If the Samples Operator detects it is on an IPv6 network.
+* If global proxy is configured.
+* If the Samples Operator detects it is installed on IBM Z.
```

Comment:
@gabemontero is this sufficient for ppc64le or S390x? Or should I list them explicitly?

Reply:
as discussed in slack list explicitly